CHANGELOG/CHANGELOG-v0.1.0.md

```diff
@@ -0,0 +1,26 @@
+# Changelog since [v0.0.6](https://github.com/Edgenesis/shifu/releases/tag/v0.0.6)
+
```

Comment:
Need to add ⚠️ Breaking change and put PR https://github.com/Edgenesis/shifu/pull/217 under it, same for Chinese